GltfExport.cs

```diff
@@ -1,16 +1,20 @@
-﻿using Grasshopper.GUI;
-using Grasshopper.Kernel;
+﻿using Grasshopper.Kernel;
 using Grasshopper.Kernel.Data;
 using Grasshopper.Kernel.Types;
 using Rhino;
+using Rhino.DocObjects;
 using Rhino.FileIO;
 using Rhino.Geometry;
+using Rhino.Render;
+using Rhino.Render.ChangeQueue;
+using System.Collections.Concurrent;
 
 namespace Web_Exporter
 {
     public class GltfExport : GH_Component
     {
         private string _lastResult = "";
+        private bool _executing = false;
```

Comment:
# Threading/GH-Specific-Flow - Managing Concurrent Executions
 - I recognize this from the async version you made, where I attempted to summarize it in this image:
![GH_TemplateAsync_](https://github.com/timsdesigns/GLTF-Export/assets/10819086/5ec0936e-99cf-4093-b150-951de2903704)
 - Maybe down the line we could use methods for that, e.g.;
 ```csharp
 private void StartExportProcess()
{
    if (_executing) return;
    _executing = true;
    // Proceed with the export process...
}

private void EndExportProcess()
{
    _executing = false;
}
```